ROMFS/px4fmu_common/init.d/airframes/4400_ssrc_fog_x

```diff
@@ -26,6 +26,9 @@ param set-default MC_PITCHRATE_I 0.3
 param set-default MC_ROLLRATE_D 0.004
 param set-default MC_PITCHRATE_D 0.004
 
+# Increase velocity controller P gain
+param set-default MPC_XY_VEL_P_ACC 2.4
```

Comment:
Using 2.4 for this parameter causes very aggressive motions.
Also, in some cases, causes overshooting accelerations that requires slowing down. When this happens, drone cannot follow a mission smoothly, causing drone accelerate and de-accelerate repeatedly.

We observed that using 1.8-1.9 actually provides much smoother flights.

Reply:
We did multiple flights outdoors, checking the logs, and observed that the drone velocity tracks the velocity setpoint much better with higher value. So I suppose smoothness which you are observing is caused by drone trying to change velocity, but in reality the velocity doesn't change as fast as supposed. This does make flying smooth, but also like the drone had too many Peronis .... To make it really smoother, instead try to decrease MPC_XY_VEL_P and/or MPC_XY_P. This would result it *not to try* to change velocity that fast. :)

To see how lazy velocity tracking may affect see this log:
https://px4-flight-review.ssrc.fi/plot_app?log=ce42439b-46af-4b45-bd7e-caa8ff4eeede

Observe x and y position, and then velocity x and y vs the setpoints. The true velocity lagging behind the setpoint makes is oscillate around the position/trajectory, since it is always flying at wrong speed vs. what it tries to do.

Reply:
That is true. The behavior change that I mentioned was with the indoor flights. Probably, it was the difference of the positioning accuracy between indoor lighthouse and the GPS.